src/VisualStudio/Core/Def/Options/VisualStudioOptionStorage.cs

```diff
@@ -385,7 +393,7 @@ public bool TryFetch(LocalUserRegistryOptionPersister persister, OptionKey2 opti
         {"SolutionCrawlerOptionsStorage_BackgroundAnalysisScopeOption", new RoamingProfileStorage("TextEditor.%LANGUAGE%.Specific.BackgroundAnalysisScopeOption")},
         {"SolutionCrawlerOptionsStorage_CompilerDiagnosticsScopeOption", new RoamingProfileStorage("TextEditor.%LANGUAGE%.Specific.CompilerDiagnosticsScopeOption")},
         {"SplitCommentOptions_Enabled", new RoamingProfileStorage("TextEditor.%LANGUAGE%.Specific.SplitComments")},
-        {"SplitStringLiteralOptions_Enabled", new RoamingProfileStorage("TextEditor.%LANGUAGE%.Specific.SplitStringLiterals")},
+        {"SplitStringLiteralOptions_Enabled", new RoamingProfileStorage("TextEditor.CSharp.Specific.SplitStringLiterals")},
```

Comment:
Another break. This option is only used by C# and declared as `Option2<T>`, so it does have `%LANGUAGE%` substituted to "".